testhelper/certificatetest.go

```diff
@@ -139,10 +159,8 @@ func getCertTemplate(isRoot bool, cn string) *x509.Certificate {
 		template.MaxPathLen = 1
 		template.IsCA = true
 	} else {
-		template.SerialNumber = big.NewInt(2)
+		template.SerialNumber = big.NewInt(int64(mrand.Intn(200)))
 		template.NotAfter = time.Now().AddDate(0, 0, 1)
-		template.KeyUsage = x509.KeyUsageDigitalSignature
-		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageCodeSigning}
```

Comment:
Why are we removing these two lines?

Reply:
KU and EKU are already added in line 151 and 152